### PR TITLE
Release Notes 4 Nov 2015

### PR DESCRIPTION
**Pull request by lamagnifica**
## [DOC-2374](https://openedx.atlassian.net/browse/DOC-2374)

they’re back and better than ever
### Date Needed

4 Nov
### Reviewers
- [x] Subject matter expert: @shnayder
- [x] Doc team review: @srpearce 
### Testing
- [x] Ran ./run_tests.sh without warnings or errors
### Post-review
- [x] Squash commits

### CODE REVIEW

File: en_us/release_notes/source/2015/documentation/doc_1104_2015.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Building and Running an edX Course
+**********************************
+
+* Added the :ref:`partnercoursestaff:Offering Academic Course Credit` section,
+  including topics about
+  :ref:`proctored exams<partnercoursestaff:CA_ProctoredExams>`.
+
+* Added the :ref:`partnercoursestaff:Timed Exams` section.
+
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/documentation/doc_1104_2015.rst]: Add Insights

File: en_us/release_notes/source/2015/documentation/doc_1104_2015.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+Documentation for Open edX Users
+==================================
+
+This section lists updates to the *latest* versions of Open edX documentation.
+EdX does not update the documentation for named Open edX releases.
+
+Building and Running an Open edX Course
+****************************************
+
+Added the :ref:`opencoursestaff:Timed Exams` section.
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/documentation/doc_1104_2015.rst]: Add developers guide (after merge Xblock/Oppia PR)

**Comment by shnayder** [comment]
:+1:  on the analytics note and screenshot. Thanks!

**Comment by srpearce** [comment]
:+1:

**Pull request merged**